girder/models/folder.py

```diff
@@ -801,7 +801,7 @@ def copyFolderComponents(self, srcFolder, newFolder, creator, progress,
 
         # copy metadata and other extension values
         updated = False
-        if srcFolder['meta']:
+        if srcFolder.get('meta'):
```

Comment:
Should this be `if 'meta' in srcFolder:` so that empty meta dictionaries are copied?

Reply:
I specifically chose this way. I see no reason to copy an empty field, but maybe I'm missing an important case.

Reply:
I don't know of any use where having an empty meta dictionary would be treated distinctly from not having a meta field.

Reply:
I agree with @manthey , given that this is intended to copy the resource, it should copy it as faithfully as possible, by preserving all keys even if their values are falsy.

Reply:
In that case, then I should also remove the `updated` flag, since it will always get set to `True` for non-legacy `srcFolder` documents (as they always have an existent `meta` field, which will be faithfully copied).